pkg/tasks/tasks.go

```diff
@@ -0,0 +1,23 @@
+package tasks
+
+import "github.com/giantswarm/e2e-harness/pkg/harness"
+
+// Task represent a generic step in a pipeline
+type Task func(*harness.Status) (*harness.Status, error)
```

Comment:
I don't see a point in passing pointer when the status is returned back.

Reply:
Done